Packs/FeedCyrenThreatInDepth/pack_metadata.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "Cyren Threat InDepth Threat Intelligence",
+    "description": "Threat InDepth's actionable and contextualized intelligence helps enterprises improve their threat detection and response by providing unprecedented visibility into new email-borne security threats faster than other security vendors.",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Cyren",
+    "url": "https://www.cyren.com/products/threat-indepth",
+    "email": "christian.meilke@cyren.com",
```

Comment:
This is the contact email for the pack - would you want to change it to something else? e.g. a general company info email

Reply:
Good catch, will bring this up.

Reply:
Christian, per our chat, support@ or something equiv is needed here.

Reply:
I now received a new email address from our people to put in there.